test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/UtilVerbs.java

```diff
@@ -441,15 +445,27 @@ public static HapiSpecOperation reduceFeeFor(HederaFunctionality function,
 
 	private static void reduceFeeComponentsFor(FeeSchedule.Builder feeSchedule, HederaFunctionality function,
```

Comment:
not sure that this will work out. In the previous implementation, they are updating the  `feeSchedule` object and setting the max variables to the provided arguments. You are not doing that, you are creating a new Map that's not used anywhere after that. My guess would be that we should be iterating through the feeDataBuilder list and setting the `setMax` to the `nodedata`, `network` and `service` builders